### DISCUSSION TITLE
So, no HPOS in v8?

### DISCUSSION THREAD

**Discussion by luboslives** [woocommerce/woocommerce#39880] [WooCommerce HPOS]
I just updated to version 8 and have been looking around, and High Performance Order Storage has not been fully integrated yet? The idea was to [launch with v8 of woocommerce](https://developer.woocommerce.com/roadmap/high-performance-order-storage/), and I'm seeing on the roadmap that it's [still not in the _Launched_ category](https://developer.woocommerce.com/roadmap/).

Does anyone know when it officially launches or gets mainlined? I figured it would be attached to a major version update.

**Comment by vedanshujain** [comment]
Hey, yes, we delayed the rollout to 8.2, see this post for details: [HPOS: full rollout delayed](https://developer.woocommerce.com/2023/08/04/hpos-full-rollout-delayed/).
